context.tf

```diff
@@ -19,7 +19,7 @@
 #
 
 module "this" {
-  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.22.0"
```

Comment:
@dmattia 
Please use this format 
```
  source  = "cloudposse/label/null"
  version = "0.22.0"
```

we are updating all modules now.
thanks